### ISSUE TITLE
ios13 not supported

### ISSUE THREAD

**Issue by chenqb** [iOSForensics/pymobiledevice#23]
**socket receive is null.**

![image](https://user-images.githubusercontent.com/13580767/67063126-2cc5e300-f198-11e9-8fb6-befffc35eeb8.png)

**Comment by GotoHack** [comment]
If you have installed pymobiledevice through a package manager (from your Linux or BSD system, from PyPI, etc.), please get and install the current development code, and check that the bug still exists.
If yes, can you explain, if possible, the API you are using (e.g., give examples of function calls, etc.).
Can you also details on witch OS/plateform you are running your code ?

**Comment by sabrina261087** [comment]
Tous les services déclarés dans les classes comme 'com.apple.mobile.screenshotr' sous le package pymobiledevice ne sont pas lancés sous ios 13 , et ils sont lancés sous ios 12 ,

sous ios 13 je vois l'erreur NoneType got no item

**Comment by sabrina261087** [comment]
DLMessageVersionExchange = self.service.recvPlist()
 returns an exception pour ios13

**Comment by tech234a** [comment]
I am also having a similar error when trying to initiate a backup on iOS 13.3. My system is running Python 3.8.1 x64 on Windows 10 x64. I am using the current development code, with the revisions from #27.

```python
INFO:pymobiledevice.plist_service:Connecting to device: [id]
INFO:pymobiledevice.lockdown:Using iTunes pair record: [id].plist
INFO:__main__:Starting new com.apple.mobilebackup2 service with working dir: backups
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python38\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Program Files\Python38\lib\site-packages\pymobiledevice-1.2.1-py3.8.egg\pymobiledevice\mobilebackup2.py", line 535, in <module>
    mb = MobileBackup2(lockdown, options.path)
  File "C:\Program Files\Python38\lib\site-packages\pymobiledevice-1.2.1-py3.8.egg\pymobiledevice\mobilebackup2.py", line 77, in __init__
    self.start()
  File "C:\Program Files\Python38\lib\site-packages\pymobiledevice-1.2.1-py3.8.egg\pymobiledevice\mobilebackup2.py", line 96, in start
    version_major = DLMessageVersionExchange[1]
TypeError: 'NoneType' object is not subscriptable
```

**Comment by tech234a** [comment]
Regular `libimobiledevice` is working, though it sometimes needs to be `unpair`ed and re`pair`ed.

**Comment by tech234a** [comment]
Possibly related?: https://github.com/libimobiledevice/libimobiledevice/commit/2332655423c1616d8e37ece7f33e98be0e218504

**Comment by JerryGinger** [comment]
@tech234a @chenqb  [this pymobiledevice version](https://github.com/qtacore/pymobiledevice) supported iOS 13, you can try it.

**Comment by GotoHack** [comment]
Fixed by 33e8c96af2fd7bec7b7fb33ea885eba4fa1b70c7
